cpp/chrono-include.rs

```diff
@@ -0,0 +1,7 @@
+mod swig_foreign_types_map {}
```

Comment:
Our other `chrono-include.rs` currently lives in `common`. Do you think we should use compilation flags and combine them? Or move the java one out of common and put it in each workspace member?

Reply:
It's shared there. I was assuming that common was a place for things shared between the 2 java projects. I guess we could do a cfg_if in there. Do you think that's better?

Reply:
I was thinking that `common` would be for anything we could share between members, not just between java members. I think that we should probably share `chrono-include.rs` with cpp as well, just so it's easier to know where to go looking for it

Reply:
Hmmm, it didn't seem to work to just do a cfg_if block in that file. If you want to see what I did I committed what I thought should work and then reverted. https://github.com/IronCoreLabs/ironoxide-java/pull/109/commits/b665dce281a6f0db14e8e531de0bbd1e31e8ec66

Reply:
Checked it out and played with it. It seems that `foreign_typemap!(...)` works, but `{ foreign_typemap(...) }` fails. So I'd guess that the `cfg_if` is putting in some brackets it doesn't like?
Unfortunately, I couldn't make the conditional compilation work without being able to chunk it with brackets. So if you're happy where it is, I'm fine with that